src/main/java/com/laserfiche/repository/api/clients/params/ParametersForCancelOrCloseSearch.java

```diff
@@ -12,6 +15,12 @@ public class ParametersForCancelOrCloseSearch {
      */
     private String searchToken;
 
+    /**
+     * Builder for setting the repoId parameter.
```

Comment:
maybe better to use "attribute" instead of "parameter", i.e.
Builder for setting the repoId attribute.

Reply:
I used the term "parameter" to follow the class name since it is like "ParametersFor..."

Reply:
Correct. I got it.